homeassistant/components/device_tracker/nmap_tracker.py

```diff
@@ -25,6 +25,8 @@
 CONF_EXCLUDE = 'exclude'
 # Interval in minutes to exclude devices from a scan while they are home
 CONF_HOME_INTERVAL = 'home_interval'
+CONF_OPTIONS = 'scan_options'
+DEFAULT_OPTIONS = '-F --host-timeout 5s '
```

Comment:
I stand corrected then, but still think to require spaces to be added by a user at the beginning/end in config is not ideal